cmd/gen/workflows/runner.go

```diff
@@ -76,6 +76,10 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 		inputs = append(inputs, workflowsInput.AddCustomerBoardAutomation())
 	}
 
+	if r.flag.Flavours.Contains(gen.FlavourClusterApp) {
```

Comment:
needs documentation in `gen workflows` help

Reply:
done